### PR TITLE
Add an upsample2d operator as experimental

### PR DESCRIPTION
**Pull request by houseroad**
This is experimental.

### CODE REVIEW

File: onnx/defs/experiments/defs.cc [hunk 1]
```diff
@@ -307,3 +307,29 @@ OPERATOR_SCHEMA(ATen)
...
 and op)DOC");
+
+OPERATOR_SCHEMA(ResizeNearest)
+    .SetSupportLevel(SupportType::EXPERIMENTAL)
+    .NumInputs(1)
+    .NumOutputs(1)
+    .Attr(
+        "width_scale",
+        "The scale along width dimension",
+        AttrType::FLOAT)
```
Comment (bddppq) [inline onnx/defs/experiments/defs.cc]: Please either mark the attribute as required or document the default value.

File: onnx/defs/experiments/defs.cc [hunk 2]
```diff
@@ -307,3 +307,29 @@ OPERATOR_SCHEMA(ATen)
...
+    .Input(
+        1,
+        "X",
+        "4D tensor, N x C x H x W")
+    .Output(
+        1,
+        "Y",
+        "4D tensor after resizing, N x C x H x W")
+    .SetDoc(R"DOC(
+Resize the width and height dimensions:
```
Comment (bddppq) [inline onnx/defs/experiments/defs.cc]: Maybe add some more description? It's not clear what values of the new pixels in the output.

File: onnx/defs/experiments/defs.cc (hunk 1 above)
Comment (houseroad) [reply onnx/defs/experiments/defs.cc]: Sure

File: onnx/defs/experiments/defs.cc (hunk 2 above)
Comment (houseroad) [reply onnx/defs/experiments/defs.cc]: Will add more description, and an example here.

**Comment by bddppq** [comment]
LGTM

@ebarsoum

**Comment by silaskuo** [comment]
Hello, do you have ETA to move this op from experimental to default. Thanks

**Comment by houseroad** [comment]
@silaskuo right on time, I will move it soon. PR is under construction.

**Comment by silaskuo** [comment]
Is it moved from experimental to default? If yes, then is https://github.com/onnx/onnx/blob/master/docs/Operators.md going to be updated accordingly? Thanks.

**Pull request merged**